File: Domain/AppUser.cs
using Base.Domain;

namespace Domain;

public class AppUser : BaseEntityWithMetaConcurrency
{
    public string Email { get; set; } = default!;
    public string Username { get; set; } = default!;
    public string FullName { get; set; } = default!;
    public string Locale { get; set; } = "et";
    public string? BankIban { get; set; }
}

File: frontend/src/types/appUser.ts
// Matches backend DTO/Web/AppUserDto.cs (BaseEntityWithConcurrency + fields).
// The frontend only ever talks to the Web DTO shape, never backend Domain entities.
export interface AppUser {
  id: string
  /** Concurrency token round-tripped as If-Match on updates (unused in this change). */
  concurrencyToken: string
  email: string
  username: string
  fullName: string
  locale: string
  sendNotifications: boolean
  notificationEnvironmentId: string | null
}
